src/test/java/uk/nhs/hee/tis/trainee/sync/facade/PlacementEnricherFacadeTest.java

```diff
@@ -107,7 +107,6 @@ void shouldEnrichFromPlacementWhenPostAndSameTrustsExist() {
 
```

Comment:
Revert the changes to this class, we should let the enricher take care of when/what to request so these verifications will be needed again.